blender_3dmigoto.py

```diff
@@ -2600,11 +2605,17 @@ def invoke(self, context, event):
 
     def execute(self, context):
         try:
+            #Check if the new mod folder is inside "Mods"
+            if os.path.basename(os.path.dirname(os.path.dirname(self.filepath))).lower() != "mods":
+                self.report({'ERROR'}, "You must export into a folder created inside the Mods folder.")
+                return self.invoke(context, None)
```

Comment:
Why not instead have the path to the ShaderOverrides folder as something that can be specified in the export dialog? You can set it to a default value like ..\ShaderOverrides.

However I'm still not convinced this is the best solution, because it's going to make it harder to correctly distribute a mod (modder has to remember which ShaderOverrides files they need to include) and to install it (users may fail to extract everything to the correct folders).

This could be a decent stop-gap measure until I get around to adding the proposed changes I mentioned last time to 3DMigoto (plus forward porting them to geo-11 and geo-12... this will take time) - it does have the advantage that it will work today and won't need users to update 3DMigoto, so I'm not entirely opposed to the idea, just need to make sure we are happy with it.

Reply:
Since I added a new option on export to generate the .ini file, this restriction will now only apply when generating the ini files. 

I didn't see a reason why someone would not put their .ini mod inside the mods folder, but I can remove it if you still prefer 

I also moved ShaderOverrides to now be in Mods/_ShaderOverrides

Reply:
Forgot to mention, modders should know which shaders they need to include by looking at the .ini itself, which has both pixel shader and vertex shader commented as shown in first screenshot